.npmignore

```diff
@@ -28,6 +28,7 @@
 /tests/
 /yarn-error.log
 /yarn.lock
+/pnpm-lock.yaml
```

Comment:
I guess we don't want to ignore this one but `package-lock.json`

Reply:
It's in `.npmignore` which excludes files from publishing on npm registry. We don't need `pnpm-lock.yaml` there, right?

Reply:
Oh, my bad, didn't pay attention to the file name 🙈